tests/app/views/handlers/test_confirmation_email_fulfilment_request.py

```diff
@@ -8,22 +8,25 @@
 
 @freeze_time(time_to_freeze)
 def test_confirmation_email_fulfilment_request_message(
-    session_data, confirmation_email_fulfilment_schema
+    fake_questionnaire_store, confirmation_email_fulfilment_schema
 ):
```

Comment:
This test doesn't seem to use `questionnaire_store` so just the metadata needs to be passed in as the fixture.